### ISSUE TITLE
Bad timings on the server.

### ISSUE THREAD

**Issue by Mr-Krab** [CoFH/Feedback#1103]
If the players will be more than 15, then I will have problems.

`
##### # lscpu
Architecture:        x86_64
CPU op-mode(s):      32-bit, 64-bit
Byte Order:          Little Endian
CPU(s):              4
On-line CPU(s) list: 0-3
Thread(s) per core:  1
Core(s) per socket:  4
Socket(s):           1
NUMA node(s):        1
Vendor ID:           GenuineIntel
CPU family:          15
Model:               6
Model name:          Common KVM processor
Stepping:            1
CPU MHz:             3292.374
BogoMIPS:            6584.74
Hypervisor vendor:   KVM
Virtualization type: full
L1d cache:           32K
L1i cache:           32K
L2 cache:            4096K
L3 cache:            16384K
NUMA node0 CPU(s):   0-3
Flags:               fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx lm constant_tsc nopl xtopology cpuid pni cx16 x2apic hypervisor lahf_lm pti

`

**Comment by KingLemming** [comment]
No. This is an artifact of how profiling works. The Tick Handler gets "blamed" for literally everything connected to a duct.
